packages/server/src/Server.ts

```diff
@@ -3,17 +3,13 @@ import {
   createServer,
 } from 'http'
 import type { ListenOptions } from 'net'
-import { fileURLToPath } from 'node:url'
-import { dirname, join } from 'node:path'
-import fs from 'node:fs'
 import type WebSocket from 'ws'
 import { WebSocketServer  } from 'ws'
 import type { AddressInfo, ServerOptions } from 'ws'
 import kleur from 'kleur'
 import { defaultConfiguration, Hocuspocus } from './Hocuspocus.js'
 import type { Configuration, onListenPayload } from './types'
-
-const meta = JSON.parse(fs.readFileSync(join(dirname(fileURLToPath(import.meta.url)), '../package.json'), 'utf-8'))
+import meta from '../package.json' assert { type: 'json' }
```

Comment:
This will pull the package.json at build time, so no need to look for a package.json at runtime.

It even does tree-shaking to just the version that we are using

Reply:
Hm, so I just removed ts-node (new Node versions can actually run typescript with a flag now: https://nodejs.org/en/learn/typescript/run-natively)

& it seems to be running for me now.